src/ToolWindowManager.h

```diff
@@ -343,7 +343,7 @@ class ToolWindowManager : public QWidget
    * \brief Creates new area and sets its default properties. You may reimplement
    * this function to change properties of all tab widgets used by this class.
    */
-  virtual ToolWindowManagerArea *createArea();
+  virtual ToolWindowManagerArea *createArea(QWidget *owner = nullptr);
```

Comment:
Minor style nit - `NULL` rather than `nullptr`.

Reply:
Apologies, forgot to check which you were using.